.muse/install_eslint_plugins.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+npm install @typescript-eslint/eslint-plugin@latest --save-dev
```

Comment:
Is the issue here that it doesn't like that our `package.json`s aren't in the top-level directory?

Reply:
Not really sure.  I was just looking at the logs at https://console.muse.dev/result/sonatype/sonatype-react-shared-components/01F1097FRGAF2MSMP8JJAW6RKE?tab=logs and trying to play with things a little without necessarily bugging the muse.dev folks too much.

The analysis for this PR didn't have a build log.  Maybe because I didn't change any typescript.  I may try to introduce an error.